charts/nublado2/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
```

Comment:
This file seems pretty useless. Maybe just drop it? I never found it did anything useful for Gafaelfawr.